stubs/ruamel.yaml/METADATA.toml

```
@@ -0,0 +1,5 @@
+version = "0.18.*"
+upstream_repository = "https://sourceforge.net/p/ruamel-yaml/code/"
+
+[tool.stubtest]
+stubtest_requirements = ["configobj"]
```

It seems that stubtest/mypy can't parse `ruamel/yaml/main.py` from the upstream package, so we need to disable stubtest:

```suggestion
[tool.stubtest]
skip = true  # mypy parse errors in upstream package
stubtest_requirements = ["configobj"]
```